packages/strapi-admin/ee/controllers/role.js

```diff
@@ -3,11 +3,11 @@
 const { yup, formatYupErrors } = require('strapi-utils');
 const {
   validateRoleCreateInput,
-  validateRoleUpdateInput,
-  validateRolesDeleteInput,
   validateRoleDeleteInput,
+  validateRolesDeleteInput,
 } = require('../validation/role');
 const { validatedUpdatePermissionsInput } = require('../validation/permission');
+const { validateRoleUpdateInput } = require('../../validation/role');
```

Comment:
You can remove this line and remove the whole function at line 35. It is the same as in CE